app/views/results/marker/_overall_remark_comment.html.erb

```diff
@@ -10,12 +10,29 @@
 		    cols: 50,
 		    rows: 5,
 		    id: 'overall_remark_comment_text_area',
-		    onkeypress: "$('overall_remark_comment_submit').enable();" %>
+		    onkeypress: "jQuery('#overall_remark_comment_submit').prop('disabled',false);" %>
     <div>
       <%= f.submit t(:save_changes),
-      	           disable_with: t('working'),
-		   disabled: true,
-		   id: 'overall_remark_comment_submit' %>
+                   disabled: "disabled",
+                   id: 'overall_remark_comment_submit' %>
     </div>
   <% end -%>
 </div>
+<script>
```

Comment:
The jQuery objects should be different (different ids), so I don't think there should be any overriding. But this brings up another point - can you actually move both code blocks to _boot.js?